torch/_torch_docs.py

```diff
@@ -3892,6 +3892,45 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins, *, range=None, weight=None, density=False, out=None) -> (Tensor, Tensor)
```

Comment:
Given that this doesn't document min and max, should they just be removed?

Reply:
I think the main upside of secretly supporting `min` / `max` is that `histogram` can be dropped in for `histc` seamlessly. Would that be worth keeping them as hidden kwargs?